Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Registration.hs

```diff
@@ -110,7 +114,7 @@ login LoginReq {..} = do
     throwError $ InvalidRequest "Server for this merchant is not available"
```

Comment:
```
merchant <- QMerchant.findByShortId merchantId >>= fromMaybeM (MerchantDoesNotExist merchantId.getShortId)
```

Instead of just merchant check for both merchnatShortId and city if merchant city entry exist for that.

Reply:
Didn't understand what to do, can we connect for it?